src/OpenTelemetry.AutoInstrumentation.Logging/OpenTelemetry.AutoInstrumentation.Logging.csproj

```diff
@@ -0,0 +1,11 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <Import Project="..\CommonExcludedAssets.props" />
+
+  <PropertyGroup>
+    <Version>0.5.1</Version>
```

Comment:
Option 1: Update `releasing.md` with info to bump the version to.

Option 2: Maybe we can mve this property to `src/Directory.Build.props`?

Reply:
It might be simpler to use option 1 for now, and perhaps try out option 2 separately.

Reply:
I have updated `releasing.md` for now.